src/main/java/org/junit/internal/ServiceLoaderWrapper.java

```diff
@@ -0,0 +1,73 @@
+package org.junit.internal;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.ServiceConfigurationError;
+import java.util.ServiceLoader;
+
+/**
+ * Load services by specified service type using Java SPI.
+ *
+ * @since 4.12
```

Comment:
Actually, the main branch is 4.13. JUnit 5 changes are on the java5 branch. We may decide not to release 4.13 and jump to 5.0, but for the time being we're having people say `@since 4.13` on main